src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocLexer.g4

```diff
@@ -37,8 +37,7 @@ import java.util.*;
 
 }
 
-LEADING_ASTERISK : ( (' '|'\t') {_tokenStartCharPositionInLine == 0}? ) (' '|'\t')* '*'
-      | '*' {_tokenStartCharPositionInLine == 0}?
+LEADING_ASTERISK : (' '|'\t')* '*'
```

Comment:
no, we should not do any conversions to sources, at least till clearly reported as bug with many supporters.
So for now lets keep this is as limitation.